eng/illink.targets

```
@@ -210,23 +213,15 @@
       <!-- pass the non-embedded descriptors xml file on the command line -->
       <ILLinkArgs Condition="'$(ILLinkDescriptorsLibraryBuildXml)' != ''">$(ILLinkArgs) -x "$(ILLinkDescriptorsLibraryBuildXml)"</ILLinkArgs>
       <ILLinkArgs Condition="'$(ILLinkSubstitutionsLibraryBuildXml)' != ''">$(ILLinkArgs) --substitutions "$(ILLinkSubstitutionsLibraryBuildXml)"</ILLinkArgs>
+      <ILLinkArgs Condition="'@(ILLinkSuppressionsLibraryBuildXml)' != ''">$(ILLinkArgs) --link-attributes &quot;@(ILLinkSuppressionsLibraryBuildXml->'%(FullPath)', '&quot; --link-attributes &quot;')&quot;</ILLinkArgs>
```

You shouldn't need the outer quotes here:

```suggestion
      <ILLinkArgs Condition="'@(ILLinkSuppressionsLibraryBuildXml)' != ''">$(ILLinkArgs) --link-attributes "@(ILLinkSuppressionsLibraryBuildXml->'%(FullPath)', '&quot; --link-attributes &quot;')"</ILLinkArgs>
```

I'm not even sure about the inner quotes. `&quot;` is usually only required in item includes: `<Item Include="&quot;...&quot;" />`